.github/workflows/release.yml

```diff
@@ -30,9 +31,23 @@ jobs:
         HUGO_PARAMS_SEARCH_APIKEY: ${{ secrets.HUGO_PARAMS_SEARCH_APIKEY }}
         HUGO_PARAMS_SEARCH_ENABLED: true
         HUGO_PARAMS_SEARCH_INDEX: thethingsstack
-    - name: Deploy documentation to Github pages
-      uses: JamesIves/github-pages-deploy-action@releases/v3
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
       with:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        BRANCH: gh-pages
-        FOLDER: public
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ secrets.AWS_REGION }}
+    - name: Build docker image
+      run: sudo docker image build -t thethingsindustries/thethingsindustries:docs .
+    - name: Docker Login
+      run: docker login -u ${{ secrets.DOCKERHUB_USERNAME }} -p ${{ secrets.DOCKERHUB_TOKEN }}
+    - name: Push docker image
+      run: docker push thethingsindustries/thethingsindustries:docs
+    - name: Restart ECS
+      run: |
+        aws ecs update-service --cluster docs-cluster --service docs-dev --force-new-deployment
+        aws ecs update-service --cluster docs-cluster --service docs-prod --force-new-deployment
+    - name: Wait for steady state
+      run: |
+        aws ecs wait services-stable --cluster docs-cluster --service docs-dev
+        aws ecs wait services-stable --cluster docs-cluster --service docs-prod
```

Comment:
I don't like this at all. We shouldn't deploy things to production by relying on side-effects. Instead, I think we should push a Docker image with a tag consisting of (a date and) a commit hash and then update the deployment to use that new tag.

Reply:
Why? How can this fail?

Reply:
If the deployment fails because the container crashes, ECS won't be able to rollback to the old image, because there is no old image (tag) anymore.

Reply:
That's not true.

If new image keeps failing to start, ECS will keep restarting it while routing traffic to tasks with old image.

If new image starts, and after some time gets killed (for example by load balancer), then service will be unhealthy, ECS will keep restarting it. No rollback.